docs/source/example_notebooks/test_data.csv

```diff
@@ -0,0 +1,5001 @@
+X0,Z0,Z1,W0,W1,W2,W3,W4,v0,y
```

Comment:
can you create this file inside a folder, called `datasets` within `example_notebooks`?

Reply:
Actually, is there a need to add this csv file to the github repo? I guess the user can create it themselves when they run the code in the notebook?

Reply:
I added this so that the notebook can run correctly while testing. If that's not necessary, it can be removed